### PR TITLE
use pipenv to manage environment

### PR DESCRIPTION
**Pull request by joegilley**

### CODE REVIEW

File: DOCS.md [hunk 1]
```diff
@@ -45,11 +44,11 @@ Sample records:
...
- Row | decision_id | decision |              metrics             |   experiment_id   | mdp_id |     ts     |
------|-------------|----------|----------------------------------|-------------------|--------|------------|
-  1  |   c2aa520f  |  female  |        {"height": 158.462}       | height-prediction |    a   | 1585613418 |
-  2  |   f3e637a5  |   male   |        {"height": 172.331}       | height-prediction |    b   | 1585934016 |
-  3  |             |          | {"female": {"delayedMetric": 1}} | height-prediction |    a   | 1599999016 |
+ | Row | decision_id | decision | metrics                          | experiment_id     | mdp_id | ts         |
+ | --- | ----------- | -------- | -------------------------------- | ----------------- | ------ | ---------- |
+ | 1   | c2aa520f    | female   | {"height": 158.462}              | height-prediction | a      | 1585613418 |
+ | 2   | f3e637a5    | male     | {"height": 172.331}              | height-prediction | b      | 1585934016 |
+ | 3   |             |          | {"female": {"delayedMetric": 1}} | height-prediction | a      | 1599999016 |
```
Comment (econti) [inline DOCS.md:51]: `ocd++`

Comment (joegilley) [reply DOCS.md:51]: LOL.  I have a markdown table formatter.  can you imagine editing this by hand jfc

**Comment by econti** [comment]
test failure is probably a flaky test, testing ML models is hard...

**Comment by joegilley** [comment]
> test failure is probably a flaky test, testing ML models is hard...

oh! thanks, I was so confused trying to sort this one out

**Comment by joegilley** [comment]
heads up @econti - it was the latest version of scikit-learn (0.23) that broke tests.  I locked it back down to the original requirements version and its all good now.

**Pull request merged**